src/main/java/com/sparta/popupstore/domain/popupstore/controller/PopupStoreController.java

```diff
@@ -100,4 +101,10 @@ public ResponseEntity<Void> deletePopupStoreAdmin(
     public ResponseEntity<List<PopupStoreFindOneResponseDto>> getPopupStoreAll() {
         return ResponseEntity.ok(popupStoreService.getPopupStoreAll());
     }
+
+    @Operation(summary = "카카오 주소 API", description = "주소 기준 위도/경도 구하기")
+    @GetMapping("/popupstores/kakaoapi/{address}")
+    public KakaoSearchApiDto getKakaoAddress(@PathVariable String address) {
```

Comment:
```suggestion
    public KakaoSearchApiDto getKakaoAddress(@RequsetParam String address) {
```
PathVariable 보다는 RequsetParam 으로 받는 게 좋을 것 같습니다.